### PR TITLE
Creating a playbook for dependencies.sh

### PR DESCRIPTION
**Pull request by shivin7**
## Description
Created a `dependencies.yml` for the changes mentioned in `dependencies.sh`

## Motivation
With regard to issue [#288](https://github.com/AutolabJS/AutolabJS/issues/288)

 ## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checks
 - [x] Squash your commits down to one, or a few, discreet changesets before submitting a pull request
 - [ ] Read the [contributing guide](https://github.com/AutolabJS/AutolabJS/blob/master/CONTRIBUTING.md)
 - [x] PR should be motivated, i.e. what does it fix, why, and if relevant how
 - [ ] If possible, include an example in the description, that could help all readers (including project members) to get a better picture of the change
 - [ ] Avoid other runtime dependencies unless abslutely necessary
 - [ ] Rebase your commit history so that each commit is meaningful and states the intention behind the change
 - [ ] Ensure that your pull request follows the [style guidelines](https://github.com/AutolabJS/AutolabJS/wiki/Coding-Standards)
 - [ ] At least one commit should mention `Fixes #<issue number>` _if relevant_

## Notify reviewers
@prasadtalasila

### CODE REVIEW

File: deploy/dependencies.yml [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  become: yes
+  tasks:
+
+    - name: Setting assume-yes as true for apt-get
+      copy:
+        content: 'APT::Get::Assume-Yes "true";'
+        dest: '/etc/apt/apt.conf.d/90forceyes' 
+
+    - name: Installing lxc, bsdtar, wget, curl and 'linux-image-extra-$(uname -r)'
+      apt: name={{item}}
```
[the remark this answers is not in the document]
Comment (shivin7) [reply deploy/dependencies.yml]: Updated the change.
I've installed ansible in the script `dependencies.sh` and called the `dependencies.yml` playbook from the script itself.

File: deploy/dependencies.yml [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    - name: Installing python-pip, libssl-dev, sshpass, libffi-dev, build-essential, python-dev
+      apt:
+        name: {{ item }}
+        update_cache: yes
+      with_items:
+        - python-pip
+        - libssl-dev
+        - sshpass
+        - libffi-dev
+        - build-essential
```
Comment (prasadtalasila) [inline deploy/dependencies.yml]: we made a mistake and gave `build-essential` twice in the shell script. You can remove `build-essential` here.

File: deploy/dependencies.sh [hunk 3]
```diff
@@ -1,19 +1,5 @@
...
-
-sudo apt update
-sudo apt install -y python-pip libssl-dev sshpass libffi-dev build-essential python-dev
-sudo pip install --upgrade pip
-sudo pip install cryptography
-sudo pip install setuptools
-sudo pip install ansible
-sudo service docker restart
+sudo apt-get install ansible
+ansible-playbook dependencies.yml
```
Comment (prasadtalasila) [inline deploy/dependencies.sh]: This line should be 
`sudo ansible-playbook -i "$1" dependencies.yml`

The "$1" would be the commandline argument for the shell script specifying the inventory file to use - either single_machine or two_machines.

File: deploy/dependencies.sh [hunk 4]
```diff
@@ -1,19 +1,5 @@
 #!/bin/bash
 
 #install docker dependecy for AUFS file system
```
Comment (prasadtalasila) [inline deploy/dependencies.sh]: This comment can be removed.

File: deploy/dependencies.sh [hunk 5]
```diff
@@ -1,19 +1,5 @@
 #!/bin/bash
```
Comment (prasadtalasila) [inline deploy/dependencies.sh]: You can add descriptive header for this shell script. Sample script header can be seen [here](https://github.com/AutolabJS/AutolabJS/blob/master/tests/test_runner.sh)

**Comment by prasadtalasila** [comment]
Improvements given in PR #293 supersede the improvements given in this PR. @shivin7 please check PR #293.

**Pull request closed without merging**